### DISCUSSION TITLE
시험 기간 관련 스케줄 조율

### DISCUSSION THREAD

**Discussion by yujin1292** [yujin1292/BreakingCodingTest#26] [General]
여러분 시험기간이 곧 다가오고 있다고 들었습니다 ~ 
12월 15일~ 21일 이 시험 주 인것같은데 스터디 스케줄을 어떻게 했으면 좋겠는지 이야기 나눴으면 좋겠어요!

지난 스터디에서는 시험 2주전 부터는 올 스탑하고 시험이 끝난 후에 진행했습니다!

시험 일정 등 개인적으로 어느정도 바쁜지 가늠이 안되어서 정하기가 쉽지않네요,,ㅠㅠ 
제가 생각한 방안으로는  두가지 정도가 있는데, 어떻게 할지 ```의견``` 댓글로 남겨주시고,  ```시험개수 등 현재 바쁨의 정도```도 댓글로 남겨주세요 !

> 
> 1. 시험 기준 2주전 올스탑
> 2. 숙제의 양을 절반 이상 줄이고 스터디는 계속 진행

@jeongiin @nawonhee @j-miiin

**Comment by jeongiin** [comment]
저는 대체 과제(프로젝트 및 발표) 2개랑 기말고사 1개 있습니다!
12월에 제안서 플젝 투입되면서 11월보단 좀 더 바빠지는데 재정비겸 12월 13,20 두번정도 쉬어가도 괜찮다고 생각합니다!
방학때 추진력을 얻기 위해서💨💨💨💨💨

**Reply by yujin1292** [reply]
음.. 딱 시험 보는 주에만 쉬어가고 싶은건가요?!
첫 시험 기준 1주일에 부터 쉬어도 괜찮을것같긴한데.. ㅠ

**Reply by jeongiin** [reply]
6, 13, 20이용???

**Reply by jeongiin** [reply]
저는 6, 13, 20 쉬어도 괜찮습니다!-!

**Comment by yujin1292** [comment]
음 12월 6일에 숙제없이 수업만 진행하는건어때요?
13 20은 그냥 잠시 시험에 집중해주시고 시험끝나고 다시만나는거죠!..

진행 방식도 조금 바꿔보려고 하는데 ㅠㅠ
혼자서 수업하는것보다는 각자 문제를 정해서 풀어오고, 풀어온 문제를 다른사람들에게 설명해주는 방식으로 진행해볼까 라는 생각도 들어요

@jeongiin @j-miiin @nawonhee

**Reply by jeongiin** [reply]
> 음 12월 6일에 숙제없이 수업만 진행하는건어때요? 13 20은 그냥 잠시 시험에 집중해주시고 시험끝나고 다시만나는거죠!..

좋습니다! 아예 쉬는 거보다 도움될 거 같아요

> 진행 방식도 조금 바꿔보려고 하는데 ㅠㅠ 혼자서 수업하는것보다는 각자 문제를 정해서 풀어오고, 풀어온 문제를 다른사람들에게 설명해주는 방식으로 진행해볼까 라는 생각도 들어요

1. 지금 예정된 진도를 빠르게 진행 후 문제 설명하는 방식으로 변경
2. 지금 예정된 진도를 현재 속도로 진행 + 문제 설명하는 시간 추가
3. 지금 예정된 진도를 느리게 진행 + 문제 설명한는 시간 추가
4. 예정된 수업 내용 진행 없이 각자 문제 설명하며 스터디 진행

중에 어떤 방식 말씀하시는 걸까요????
진도는 project에 있는 진도를 말합니다!

**Reply by yujin1292** [reply]
음.. 아마도 3번이 유력하긴한데요 아니면 돌아가면서 문제+개념을 설명해주는방식도 좋을것같긴해요! 

아마 현재방식 or 3번으로 진행한 후에 진도가 다 나가면 4번으로 진행할것같아요

**Reply by jeongiin** [reply]
4번 인당 매주 발표라면 10~15분 정도씩 준비하면 부담 없을 거 같아요!
3번이라면 갓유진언니의 강의 + 주당 1명씩 발표하면 시간 적당할 거 같슴다

**Comment by j-miiin** [comment]
저는 12월 6일에 팀프로젝트 최종 발표 영상이랑 보고서 제출해야 돼서 오늘부터 2주간 바쁠 것 같습니다..! 😢

**Reply by yujin1292** [reply]
오잉 시험기간은 괜찮고요?!

**Comment by nawonhee** [comment]
아직 정확한 시험 일정이 안 나오긴 했는데 듣는 수업이 적지는 않아서.. 위에 언급된 것처럼 6일에 숙제없이 진행하고 13일,20일은 쉬는 것도 괜찮은 것 같아요!
